changedetectionio/tests/test_ignorehyperlinks.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python3
+"""Test suite for the render/not render anchor tag content functionality"""
+
+import time
+from flask import url_for
+from .util import live_server_setup
+
+import multiprocessing
+multiprocessing.set_start_method('fork')
```

Comment:
What does this one do? whats the point of adding it when its not in the other tests?

Reply:
Have removed this. It's something I have to call to be able to run the unit test locally on my m1 mac